test/integration/MCR/getMCR.js

```diff
@@ -73,16 +102,16 @@ describe('getMCR', function () {
       const storedMCR = await mcr.mcr();
       const latestMCR = await mcr.getMCR();
 
-      const maxMCRIncrement = await mcr.maxMCRIncrement();
+      const maxMCRIncrement = BigNumber.from(await mcr.maxMCRIncrement());
       const expectedPercentageIncrease = maxMCRIncrement.mul(passedTime).div(daysToSeconds(1));
 
-      const expectedMCR = storedMCR.mul(expectedPercentageIncrease).divn(10000).add(storedMCR);
-      assert.equal(latestMCR.toString(), expectedMCR.toString());
+      // TODO: expected MCR is off here
```

Comment:
AssertionError: expected 49795000000000000000000 to equal 50205000000000000000000. The numerical values of the given "ethers.BigNumber" and "ethers.BigNumber" inputs were compared, and they differed.
      + expected - actual

      -49795000000000000000000
      +50205000000000000000000

Reply:
My guess there is something wrong in the way the MCR parameters are configured.

it's not a major difference here so something looks misconfigued

```const expectedMCR = storedMCR.mul(expectedPercentageIncrease).div(10000).add(storedMCR);```

this is the expected value which just says the MCR is changing by  a certain percentage proportional to how much time passed oout of the day up to 1% for the full day.


so if only 12h passed it changes by 0.5%.

i would console.log inside the code between each computation to see where value is off

Reply:
So I made a couple fixes and remembered that `buyCover` needs to be called to expire old covers. Only the final assertion is failing now: 
```
         MCR integration tests
           getMCR
             increases mcr by 0.4% in 2 hours and decreases by 0.4% in 2 hours it after cover expiry:

      AssertionError: expected 50300000611213038814781 to equal 50506230174300795558352. The numerical values of the given "ethers.BigNumber" and "ethers.BigNumber" inputs were compared, and they differed.
      + expected - actual

      -50300000611213038814781
      +50506230174300795558352
      
      at Context.<anonymous> (test/integration/MCR/getMCR.js:136:31)

```